website/docs/querying_data.md

```diff
@@ -208,6 +208,31 @@ And for these use cases you should test the stability first.
 | `hoodie.metadata.index.column.stats.enable` | `false` | false | Whether to enable column statistics (max/min) |
 | `hoodie.metadata.index.column.stats.column.list` | `false` | N/A | Columns(separated by comma) to collect the column statistics  |
 
+### Hudi Catalog
+A Hudi catalog can manage the tables created by Flink, table metadata is persisted to avoid redundant table creation.
```

Comment:
Could the Hudi Flink catalog manage the tables created by other engines?

Reply:
Yes, it can translate spark created table as Flink dialect.